util.c

```diff
@@ -350,15 +350,15 @@ void intremove (intnode **root, intnode *i)
 	unsigned int key = i->key;
 	intnode *n = *root;
 
-	if (!(isroot = n == i))
+	if (!((isroot = n == i))) {
```

Comment:
no need in another round brackets. Or they should be in other way: `if (!(isroot = (n == i))) {`

Reply:
For sure you are right here and I did a mistake here, thanks !